docs/cloud/alerts-notifications/add-discord-notification.md

```diff
@@ -0,0 +1,61 @@
+<!--
+title: "Add discord notification configuration"
+sidebar_label: "Add discord notification configuration"
+custom_edit_url: "https://github.com/netdata/netdata/blob/master/docs/cloud/alerts-notifications/add-discord-notification-configuration.md"
+sidebar_position: "1"
+learn_status: "Published"
+learn_topic_type: "Tasks"
+learn_rel_path: "Operations/Alerts"
+learn_docs_purpose: "Instructions on how to add notification configuration for discord"
+-->
+
+From the Cloud interface, you can manage your space's notification settings and from these you can add specific configuration to get notifications delivered on discord.
+
+#### Prerequisites
+
+To add discord notification configurations you need
+
+- A Cloud account
+- Access to the space as and Administrator
+- Have your discord server able to receive webhook integrations, for mode details check [how to configure this on discord](#settings-on-discord)
```

Comment:
```suggestion
- Have a Discord server able to receive webhook integrations. For mode details check [how to configure this on discord](#settings-on-discord)
```

Reply:
will also see where discord is not capitalzed